magento/pwa-studio
# Hides the Rating component

## Description

This PR hides the `Rating` component from the product tiles on the Category page because it has not been updated to the current version of Venia look and feel.

A separate ticket (PWA-2512) will update the component and re-enable it.

## Related Issue







Closes PWA-2510.

## Acceptance





### Verification Stakeholders


@dpatil-magento 

### Specification




## Verification Steps






1. Load up an environment where products have ratings (`rating_summary` GQL field has a value)
2. Verify that the `Rating` component does not display on the product tile on the Category page

#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





No

#### Any ad-hoc/edge case scenarios that need to be considered?




No

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
